src/composables/trade/useTrading.ts

```diff
@@ -226,6 +246,14 @@ export default function useTrading(
     handleAmountChange();
   });
 
+  onMounted(() => {
+    const gaslessDisabled = window.location.href.includes('gasless=false');
```

Comment:
This is used to force the trade interface to use Balancer instead of Gnosis (for example, when copperlaunch links to us during LBP).

It's important that this happens only once (when the component mounts), so that the user can switch back to gasless.

@garethfuller This was the "bullet proof" way to ensure this query string is in the URL... Using `vue-router` doesn't quite work with `onMounted` because it won't pick up there is a query string - it will only pick it up when the URL is being watched/computed etc.

Reply:
Happy to do it this way since it's straightforward. I think it might have been possible with router guards but I'm not sure: https://next.router.vuejs.org/guide/advanced/navigation-guards.html#in-component-guards